fix: some lint fix using tools

yihong0618 opened the pull request:
This patch using some tools to make the format better

- using ruff to drop useless import
- using [typos](https://github.com/crate-ci/typos) to fix a typo in src/
- using [autocorrect](https://github.com/huacnlee/autocorrect) to lint README CN to make it beautiful
- using markdownlint to lint markdown

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/bytedance/deer-flow?pullRequest=98) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/bytedance/deer-flow?pullRequest=98) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/bytedance/deer-flow?pullRequest=98) it.</sub>

hetaoBackend commented:
That's super cool, thanks for helping lint the codebase, could u plz rebase the latest code? Thanks in advance! BTW, it might be a good idea to add ruff check in the current `make lint` command?

yihong0618 commented:
> That's super cool, thanks for helping lint the codebase, could u plz rebase the latest code? Thanks in advance! BTW, it might be a good idea to add ruff check in the current `make lint` command?

will do it tomorrow

yihong0618 commented:
> That's super cool, thanks for helping lint the codebase, could u plz rebase the latest code? Thanks in advance! BTW, it might be a good idea to add ruff check in the current `make lint` command?

@hetaoBackend addressed the comments

WillemJiang commented:
@yihong0618 Thanks for the contribution. 
Some other lint errors will be fixed in the next commit.

yihong0618 commented:
> @yihong0618 Thanks for the contribution. 
> 
> Some other lint errors will be fixed in the next commit.

thank you
